src/main/java/com/commercetools/project/sync/service/CustomObjectService.java

```diff
@@ -0,0 +1,23 @@
+package com.commercetools.project.sync.service;
+
+import com.commercetools.project.sync.model.LastSyncCustomObject;
+import io.sphere.sdk.customobjects.CustomObject;
+import java.time.ZonedDateTime;
+import java.util.Optional;
+import java.util.concurrent.CompletionStage;
+import javax.annotation.Nonnull;
+
+public interface CustomObjectService {
+  @Nonnull
+  CompletionStage<ZonedDateTime> getCurrentCtpTimestamp();
```

Comment:
I think, `getLatestTimestamp()` is better, same with the key.

Reply:
why latest? actually the purpose is to get the current (right now) timestamp of the platform.

Reply:
I thought the key for customObject is `latestTimestamp`. I think I misunderstand this method. You can ignore.

Reply:
true, the key is confusing. It needs to be changed. I will change it :)

Reply:
this has changed here: https://github.com/commercetools/commercetools-project-sync/pull/30/files#diff-90d173bf2f68b44092d0e0fa23e07f36R28